requirements.txt

```diff
@@ -8,4 +8,6 @@ flask-api
 hpOneView
 jsonschema
 pbr>=2.0 # Apache-2.0
+pika
```

Comment:
so ampq is not working at all?

Reply:
it is not working with cert auth as oneview don't accept  user auth can't tell you if with other auth mechs it would be ok